printer_mods/KevinAkaSam/VEFACH/README.md

```diff
@@ -0,0 +1,52 @@
+<img src="Images/VEFACH.png" width="300">
+
+# VEFACH - Voron Exhaust Filter Activated Coal + Hepa
+
+<img src="Images/CAD1.png" width="400"/> <img src="Images/CAD2.png" width="400"/>
+
+<img src="Images/CAD3.png" width="400"/> <img src="Images/CAD4.png" width="400"/>
+
+### Why?
+Since the printer produces many small particles, the goal is to filter out as many of them as possible. A few months ago my friend Deutherius had the idea to equip the original exhaust fan with activated coal and a Hepa filter, so I designed these filter inserts. The activated carbon removes e.g. the smell of ABS and the Hepa filter catches most of the small particles.
+ 
+### How does it work?
+The air first blows through activated carbon particles and then through the Hepa filter. This still doesn't remove all particles, but it is better than nothing. This mod is fully compatible with the stock exhaust fan housing, except original M3x30mm screws that hold the fan are too long and need to be replaced with M3x25mm. The filter materials can be removed and replaced very easily via the original cover.
+
+### How to print?
+You just need to print the two inserts for the housing. The infill is used as mesh for the activated carbon container. There are two ways to print the bucket:
+#### 1. Use the filter dummy as modifier:
+I have attached a few screenshots of how to prepare the print with the dummy filter file. The green options shown in the last two screenshots are optional. Either you set the infill of the whole bucket higher or you add another setting for the dummy FIlter. Except for the top end, the print consists of walls only. Make sure that the dummy filter and the bucket are centered. If you want to move them, then select both at the same time so you don't change the cropping area.
+
+
+1: Select Filter Dummy as modifier        |  2: Select settings for modifier
+:-------------------------:|:-------------------------:
+![](Images/1.jpg)  |  ![](Images/2.jpg)
+3: Select Perimeters Shell        |  4: Set Top-/Bottom-Layer to 0
+![](Images/3.jpg)  |  ![](Images/4.jpg)
+4: Set infill density + Type to Grid        |  5: (Optional) Set different infill density for modifier
```

Comment:
You have step 4 twice.  This last row should be steps 5 and 6, not 4 and 5.

Reply:
Hey, thanks again for your detailed feedback. I fixed all the mistakes again and added a version with mesh. Therefore I have also changed the README a bit. Sorry I made some new mistakes again, I hope this time everything is ok. Thanks for your help and patience! :)